JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -46,7 +63,83 @@ public class OAuthSampleApp implements Callable<Integer> {
 
     @Option(names = { "-s", "--client-secret" }, description = "Client Secret")
     private String clientSecret = "";
+	
```

Comment:
These are more than we need to have. We can just have the token URL be one of the properties instead of building the token URL ourselves from the different pieces. Instead of OAUTH_SERVER, OAUTH_REALM, OAUTH_HTTP_PORT we can instead just have OAUTH_TOKEN_URL.

Reply:
I agree.   The goal is to have a sample application that is closely aligned with the ADO.NET OAuth sample.   

https://github.com/vertica/client-application-examples/tree/main/ADO.NET/OAuthSampleApp

That application has a connection URL in a properties file.   The sample can be configured by editing the properties file.  No edits to the sample code are necessary.   We should do the same here.

I would like this code to be as close as possible to the ADO.NET sample.   We should only diverge if there is a language specific reason to do so.

Reply:
Removed the unnecessary methods.